forms-flow-web/src/models/EmployeeDataModel.js

```diff
@@ -0,0 +1,56 @@
+export default class EmployeeDataModel {
+  // Use this class properties names in the form builder ("Property Name" under API tab) to fill the form with the employee data.
+
+  constructor(data) {
+    this.__firstName = data?.first_name || "";
```

Comment:
I think we can remove the `__` here to simplify things for the form builders. 

Here's the list of fields that we'll need to restrict this to. I would suggest that we move the mapping to the Backend so we can only return those fields:

```
First Name
Last Name
BC Gov Email
BC Gov Phone number
Employee ID
Position/Title
Office/Work Street Address (Street Address, City, Province, Postal Code)
Department ID
Branch/Division
Ministry


Employee classification (BCHEI, PEA, Nurses, Schedule A, Management Excluded, Salaried Physician, OIC, Other)
Appointment status (Regular, Eligible Auxiliary, Full-time, Part-time)


Supervisor/Manager's name
Supervisor/Manager's email
Supervisor/Manager's position/title
```